app/Http/Controllers/User/HomeController.php

```diff
@@ -4,18 +4,21 @@
 
 use App\Repositories\Contracts\PostRepository;
 use App\Repositories\Contracts\TopicRepository;
+use App\Repositories\Contracts\UserRepository;
 use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
 
 class HomeController extends Controller
 {
     protected $topic;
     protected $post;
+    protected  $user;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ScopeKeywordSpacing.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/User/HomeController.php</br>**Line**: 15</br>Scope keyword "protected" must be followed by a single space; found 2